pkg/provision/storage/shared.go

```diff
@@ -58,18 +58,23 @@ func getCommonPVCSpec(namespace string, size string) (*corev1.PersistentVolumeCl
 	}, nil
 }
 
+// isEphemeral evaluates is volume component is configured as ephemeral
+func isEphemeral(volume dw.VolumeComponent) bool {
+	return volume.Ephemeral != nil && *volume.Ephemeral
+}
+
 // needsStorage returns true if storage will need to be provisioned for the current workspace. Note that ephemeral volumes
 // do not need to provision storage
 func needsStorage(workspace *dw.DevWorkspaceTemplateSpec) bool {
 	projectsVolumeIsEphemeral := false
 	for _, component := range workspace.Components {
 		if component.Volume != nil {
 			// If any non-ephemeral volumes are defined, we need to mount storage
-			if !component.Volume.Ephemeral {
+			if isEphemeral(*component.Volume) {
```

Comment:
```suggestion
			if !isEphemeral(*component.Volume) {
```

Reply:
I must find a good excuse for making a lot of mistakes in this simple PR ;-D